packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/create-app-integration.md

```diff
@@ -0,0 +1,24 @@
+Before integrating Okta authentication to your React app, you need to register your app in your Okta org. This provides you with the OpenID Connect client ID for authentication requests from your app. Register your app by creating an Okta app integration through the [Okta CLI](https://cli.okta.com/), the [Okta Apps API](/docs/reference/api/apps/), or through the [Admin Console](/docs/guides/quickstart/website/main/#using-the-admin-console) with the following steps.
+
+1. To create an Okta app integration that represents your React app, sign in to [your Admin Console](https://login.okta.com).
+2. From the side navigation, select **Applications** > **Applications**, and then click **Create App Integration**.
+3. In the dialog box that appears, select **OIDC - OpenID Connect** as the **Sign-on method**, **Single-Page Application** as the **Application type**, and then click **Next**.
+4. Fill in the following new app integration settings, and then click **Save**:
+
+    | Setting                | Value/Description                                    |
+    | -------------------    | ---------------------------------------------------  |
+    | App integration name   | Specify a unique name for your app                  |
+    | Grant types            | Select **Authorization Code**, **Interaction Code**, and  **Refresh Token** |
+    | Sign-in redirect URIs  | Specify your app URI for the callback redirect from Okta. For example, `http://localhost:8080/login/callback`. |
+    | Sign-out redirect URIs | Specify your app sign-out redirect URI. For example, `http://localhost:8080`. |
```

Comment:
```suggestion
    | Sign-out redirect URIs | Specify your app sign-out redirect URI. For example: `http://localhost:8080` |
```

Reply:
I added another sentence after this so retained the period.